tests/codegen/llvm/call_len_map_sum_elem_count.ll

```diff
@@ -117,5 +117,5 @@ attributes #1 = { nocallback nofree nosync nounwind willreturn memory(argmem: re
 !61 = !DISubroutineType(types: !62)
 !62 = !{!18, !63}
 !63 = !DIDerivedType(tag: DW_TAG_pointer_type, baseType: !64, size: 64)
-!64 = !DICompositeType(tag: DW_TAG_structure_type, name: "struct bpf_map", scope: !2, file: !2, elements: !65)
+!64 = !DICompositeType(tag: DW_TAG_structure_type, name: "bpf_map", scope: !2, file: !2, elements: !65)
```

Comment:
this a safe change?

Reply:
Interestingly yes, it looks like the verifier doesn't care about the name for function parameters as much as for global variables. In any case, now the name matches the name in kernel BTF.